### PR TITLE
414 add leet code icon

### PR DESCRIPTION
**Pull request by nayabatir1**
## Related Issue
* adding leetCode profile link in social media section

Fixes: #414 

**LeetCode svg is not available with fontawsome, I've added a custom svg with css**

>SocialMedia.js
```js
{socialMediaLinks.leetCode ? (
  <a
    href={socialMediaLinks.leetCode}
    target="_blank"
    rel="noopener noreferrer"
   >
     <span className="leetCode">
       <img src={leetCode} alt="leetCode" />
     </span>
    </a>
) : null}
```
>SocialMedia.scss
```scss
.leetCode {
  background-color: #fff;
  height: 40px;
  width: 40px;
  display: inline-block;
  border-radius: 50%;
  text-align: center;
  vertical-align: middle;
  box-sizing: border-box;

  img {
    vertical-align: sub;
  }
}
```

**Please, hold this PR till october**

### CODE REVIEW

File: src/assets/lottie/build.json [hunk 1]
```diff
@@ -1 +1 @@
...
...240.24024024024,"st":0,"bm":0},{"ddd":0,"ind":18,"ty":4,"nm":"Shape Layer 1","sr":1,"ks":{"o":{"a":0,"k":100,"ix":11},"r":{"a":0,"k":0,"ix":10},"p":{"a":0,"k":[926,540,0],"ix":2},"a":{"a":0,"k":[0,0,0],"ix":1},"s":{"a":0,"k":[84.211,100,100],"ix":6}},"ao":0,"shapes":[{"ty":"gr","it":[{"ty":"rc","d":1,"s":{"a":0,"k":[980,452],"ix":2},"p":{"a":0,"k":[0,0],"ix":3},"r":{"a":0,"k":20,"ix":4},"nm":"Rectangle Path 1","mn":"ADBE Vector Shape - Rect","hd":false},{"ty":"st","c":{"a":0,"k":[0.039215686275,0.039215686275,0.039215686275,1],"ix":3},"o":{"a":0,"k":100,"ix":4},"w":{"a":0,"k":12,"ix":5},"lc":1,"lj":1,"ml":4,"bm":0,"nm":"Stroke 1","mn":"ADBE Vector Graphic - Stroke","hd":false},{"ty":"fl","c":{"a":0,"k":[1,1,1,1],"ix":4},"o":{"a":0,"k":100,"ix":5},"r":1,"bm":0,"nm":"Fill 1","mn":"ADBE Vector Graphic - Fill","hd":false},{"ty":"tr","p":{"a":0,"k":[34,30],"ix":2},"a":{"a":0,"k":[0,0],"ix":1},"s":{"a":0,"k":[100,100],"ix":3},"r":{"a":0,"k":0,"ix":6},"o":{"a":0,"k":100,"ix":7},"sk":{"a":0,"k":0,"ix":4},"sa":{"a":0,"k":0,"ix":5},"nm":"Transform"}],"nm":"Rectangle 1","np":3,"cix":2,"bm":0,"ix":1,"mn":"ADBE Vector Group","hd":false}],"ip":0,"op":240.24024024024,"st":0,"bm":0}],"markers":[]}
```
Comment (kartikcho) [inline src/assets/lottie/build.json:1]: Why edit lottie width?

**Review by kartikcho — COMMENTED** [review]
Hey @nayabatir1 if fontawesome doesn't support the icon then I think we shouldn't add an extra svg for it. If a user wants to link to different profiles that aren't supported fa, they can do so themselves. 
Sorry to see this PR and you work not get merged but I highly recommend you to check out other issues!

**Comment by nayabatir1** [comment]
> Hey @nayabatir1 if fontawesome doesn't support the icon then I think we shouldn't add an extra svg for it. If a user wants to link to different profiles that aren't supported fa, they can do so themselves.
> Sorry to see this PR and you work not get merged but I highly recommend you to check out other issues!

@kartikcho 
This issue title clearly state to add leetcode profile link, isn't it?

**Comment by kartikcho** [comment]
I know it does but we didn't know that fa doesn't have a leetcode icon. 
The last similar PR added Kaggle social icon which was supported by fa so we assumed the same for this one.
Again, sorry for this and I would suggest to ask for clarifications on any other issues that you take up so your work doesn't get wasted.

**Pull request closed without merging**